bucket/main.tf

```diff
@@ -0,0 +1,44 @@
+###
+### LOCALS
+###
+
+locals {
+  name = "vault-salt-formula"
```

Comment:
I'd recommend exposing this name as a variable instead of a local, so the user can namespace it however they like